<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="suba-pc" failures="6" tests="34" name="runner.TestRunner" time="205.701" errors="0" timestamp="2024-05-02T05:30:44 EDT" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="4.644"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="34.662">
    <failure message="Assertion error: manadatory field should be enabled :programField expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Assertion error: manadatory field should be enabled :programField expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1066)
at org.testng.Assert.assertEquals(Assert.java:1030)
at pageObjects.BatchPage.checkInvalidValuesinUpdate(BatchPage.java:936)
at stepDefinitions.EditBatch_SD.erase_data_from_mandatory_field(EditBatch_SD.java:106)
at ✽.Erase data from mandatory field "AddBatch" and 1(file:///C:/Users/subas/EclipseLMSPractice/Team15_SeleniumTribe_LMSUIHackathon_Apr24/src/test/resources/Features/D_BatchModule/03_EditBatch.feature:39)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.217"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.452"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.277"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.525">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at stepDefinitions.EditBatch_SD.the_updated_batch_details_should_appear_on_the_data_table(EditBatch_SD.java:85)
at ✽.The updated batch details should appear on the data table(file:///C:/Users/subas/EclipseLMSPractice/Team15_SeleniumTribe_LMSUIHackathon_Apr24/src/test/resources/Features/D_BatchModule/03_EditBatch.feature:56)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.741"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.139"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.092"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.116"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.116"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.355"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.000">
    <failure message="Multiple rows delete icon is not deleting the batches. expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Multiple rows delete icon is not deleting the batches. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1066)
at org.testng.Assert.assertEquals(Assert.java:1030)
at stepDefinitions.DeleteBatch_SD.the_respective_row_in_the_data_table_is_deleted(DeleteBatch_SD.java:106)
at ✽.The respective row in the data table is deleted(file:///C:/Users/subas/EclipseLMSPractice/Team15_SeleniumTribe_LMSUIHackathon_Apr24/src/test/resources/Features/D_BatchModule/05_DeleteMultipleBatches.feature:29)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.175"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.465"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.096"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.882"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="36.464">
    <failure message="Multiple rows delete icon is not deleting the batches. expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Multiple rows delete icon is not deleting the batches. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1066)
at org.testng.Assert.assertEquals(Assert.java:1030)
at stepDefinitions.DeleteBatch_SD.the_respective_row_in_the_data_table_is_deleted(DeleteBatch_SD.java:106)
at ✽.The respective row in the data table is deleted(file:///C:/Users/subas/EclipseLMSPractice/Team15_SeleniumTribe_LMSUIHackathon_Apr24/src/test/resources/Features/D_BatchModule/05_DeleteMultipleBatches.feature:36)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.897"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.206"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.857"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.955"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.262"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.125"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="5.546">
    <failure message="URL does not contain &#039;manage batch&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: URL does not contain 'manage batch' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at pageObjects.BatchPage.checkURL(BatchPage.java:456)
at stepDefinitions.BatchPagevalidation_SD.admin_should_see_the_in_the_url(BatchPagevalidation_SD.java:32)
at ✽.Admin should see the "manage batch" in the url(file:///C:/Users/subas/EclipseLMSPractice/Team15_SeleniumTribe_LMSUIHackathon_Apr24/src/test/resources/Features/D_BatchModule/01_BatchPageValidation.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.874"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="31.402"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.002"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="34.487">
    <failure message="Assertion error: manadatory field should be enabled :programField expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Assertion error: manadatory field should be enabled :programField expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1066)
at org.testng.Assert.assertEquals(Assert.java:1030)
at pageObjects.BatchPage.checkInvalidValuesinUpdate(BatchPage.java:936)
at stepDefinitions.EditBatch_SD.update_the_fields_with_invalid_values_and_click_save(EditBatch_SD.java:90)
at ✽.Update the fields with invalid values and click save "AddBatch" and 0(file:///C:/Users/subas/EclipseLMSPractice/Team15_SeleniumTribe_LMSUIHackathon_Apr24/src/test/resources/Features/D_BatchModule/03_EditBatch.feature:30)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.772"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.296"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.758"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="10.817"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.027"/>
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
